src/app/qgscustomization.cpp

```diff
@@ -742,12 +763,32 @@ void QgsCustomization::updateMainWindow( QMenu *toolBarMenu )
         for ( QAction *action : constActions )
         {
           if ( action->objectName().isEmpty() )
-          {
             continue;
-          }
```

Comment:
Off topic. We should start to enforce a single style for this via astyle. Half the people add those curly braces for one liners, the other half removes them...

Reply:
Indeed. I was surprised to not get feedback from astyle.
What would be the rule? I just remembered an advice of this no-braces style from comments on a PR a while ago and followed it despite no mention is in the Dev guidelines.